lib/Perl/MinimumVersion.pm

```diff
@@ -1087,6 +1087,29 @@ sub _unquoted_versions {
 	} );
 }
 
+#Check for use re "/flags";
+sub _re_flags {
+	my ($version, $obj);
+	shift->Document->find( sub {
+		return '' unless $_[1]->isa('PPI::Statement::Include')
+			and ($_[1]->module eq 're' or $_[1]->pragma eq 're');
+		my $included = $_[1]->schild(2);
+		my @literal = $included->can('literal') ? $included->literal() : $included->string();
+		my $v = "5.005";
+		my @flags = grep {index($_, '/') == 0} @literal;
+		$v = '5.014' if scalar @flags;
+		#Check for /xx flag which was introduced in Perl 5.026
+		$v = '5.026' if grep {my $x = index($_, "x");$x > -1 and index($_, "x", $x+1) > -1} @flags;
```

Comment:
Replace with [PPIx::Regexp::Token::Modifier](https://metacpan.org/pod/PPIx::Regexp::Token::Modifier).

Reply:
This could be something like (haven't checked it)

```perl
max $v, map {
  my $empty_regex_w_flag = "/$_";
  PPIx::Regexp->new( $empty_regex_w_flag )->perl_version_introduced;
} @flags;
```

Also, could you rebase so that this runs under the CI?